modules/client/src/main/java/org/apache/ignite/internal/client/query/JdbcClientQueryEventHandler.java

```diff
@@ -56,91 +57,73 @@ public JdbcClientQueryEventHandler(TcpIgniteClient client) {
 
     /** {@inheritDoc} */
     @Override
-    public QueryExecuteResult query(QueryExecuteRequest req) {
+    public CompletableFuture<QueryExecuteResult> query(QueryExecuteRequest req) {
```

Comment:
Here and below, let's add `Async` suffix to async method names.

```suggestion
    public CompletableFuture<QueryExecuteResult> queryAsync(QueryExecuteRequest req) {
```